client/extensions/woocommerce/index.js

```diff
@@ -74,6 +76,18 @@ const getStorePages = () => {
 			documentTitle: translate( 'Product Categories' ),
 			path: '/store/products/categories/:site',
 		},
+		{
+			container: ProductCategoryUpdate,
+			configKey: 'woocommerce/extension-product-categories',
+			documentTitle: translate( 'Edit Product Category' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **106** times:
`translate( 'Edit Category' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).